src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -166,6 +167,7 @@ export class TerminalService extends Disposable implements ITerminalService {
 	@memoize get onAnyInstanceSelectionChange() { return this._register(this.createOnInstanceEvent(e => e.onDidChangeSelection)).event; }
 	@memoize get onAnyInstanceTitleChange() { return this._register(this.createOnInstanceEvent(e => e.onTitleChanged)).event; }
 	@memoize get onAnyInstanceShellTypeChanged() { return this._register(this.createOnInstanceEvent(e => Event.map(e.onDidChangeShellType, () => e))).event; }
+	@memoize get onAnyInstanceAddedCapabilityType() { return this._register(this.createOnInstanceEvent(e => e.capabilities.onDidAddCapabilityType)).event; }
```

Comment:
Did we not end up needing this one?

Reply:
weird, not sure how it got removed, we definitely need it.

Reply:
we do need that and use it here https://github.com/microsoft/vscode/pull/239742/commits/5051f7689b6061835500f7fede09a04391a68447#diff-9ad94bfccb2c181ffea0b1abb3b85b7a630784ddc3de16f47560a374da26c7ccR130

Reply:
Oh I think I was looking at a partial diff somehow